app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  after_save :update_likescounter
+
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_likescounter
+    post.increment!(:likescounter)
+  end
```

Comment:
- [OPTIONAL] As requested by the previous reviewer, make sure the `update_likescounter` is `private` 👍🏻

Reply:
Done